test/e2e/third-parties/pages/ga.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+import { GoogleAnalytics, sendGAEvent } from '@next/third-parties/google'
+
+const Page = () => {
+  const onClick = () => {
+    sendGAEvent({ event: 'buttonClicked', value: 'xyz' })
+  }
+
+  return (
+    <div class="container">
+      <GoogleAnalytics gaId="GA-XYZ" />
+      <h1>GA</h1>
+      <button id="ga-send" onClick={onClick}>
+        Click
+      </button>
+      <GoogleAnalytics gaId="GA-XYZ" />
```

Comment:
Why twice?

Reply:
This is used to test multiple initialization. We want to make sure its not duplicated